PortalFetch/crawler_download.py

```diff
@@ -204,6 +206,38 @@ def waitUtilPageLoaded(driver, count):
     raise ElementNotVisibleException("Could not load the full page!")
 
 
+def generateFilenameAndQuarter(quarterValue):
+    """Return quarter and fileName.
+
+    Args:
+        quarterValue:the quarter_value in crawler.config
+    Returns:
+        quarter str and fileName str
+
+    """
+    year = quarterValue[0:4]
+    resultList = []
+    quarterSwitcher = {
+        "1": "Summer",
+        "2": "Fall",
+        "3": "Winter",
+        "4": "Spring",
```

Comment:
nit: please be consistent with using single quotes or double quotes for the entire project